client/my-sites/email/email-forwarding/email-forwarding-add-new.jsx

```diff
@@ -121,7 +121,7 @@ class EmailForwardingAddNew extends React.Component {
 				disabled={ this.state.formSubmitting || this.hasReachedLimit() }
 				onClick={ handler }
 			>
-				{ this.props.translate( 'Add New Email Address' ) }
+				{ this.props.translate( 'Add new email address' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.